modules/banner/react/lib/Banner.tsx

```diff
@@ -16,19 +16,19 @@ export interface BannerProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    */
   onClick?: (e: React.SyntheticEvent) => void;
   /**
-   * Text inside the banner
+   * Label of the banner
    */
   label?: React.ReactNode;
   /**
-   * Size and position of the banner
+   * Set the banner variant as `full` or `sticky`
```

Comment:
Nit: Can we standardize on tenses and consistency here? i.e.
```
Label of the banner
```
vs
```
The label of the banner
```
vs
```
Set the label of the banner
```

Reply:
@anicholls Yup, let's do it! Leaning towards any of these 3?

Reply:
My vote would be a slightly modified version of the third option with `Sets` instead of `Set` (and with a period at the end):
```
Sets the label of the banner.
```

I like having the sentence work grammatically if you read the prop name immediately followed by the description (e.g., "Label sets the label of the banner." works but not "Label set the label of the banner.")

Reply:
While I agree with @jamesfan on it being nice when it works grammatically, I prefer the second option due to it's consistency across different types of props. Going with "Sets" forces us to pick different verbs for different types of props. For example, "Sets an action when interacting with the banner" is kind of a weird description for `onClick`.

I like to think about each prop as a noun, so IMO starting with "The [noun]" makes the most sense. "The action performed when...", "The text inside...", "The color of...", "The function called when...", etc.

Reply:
One exception to this actually, would be `boolean` props. I think they need to start with a verb like @jamesfan says. e.g. `Specifies/determines whether X happens`

Reply:
If we agree on some rules, it would be great to add them to https://github.com/Workday/canvas-kit/blob/master/API_PATTERN_GUIDELINES.md#props 🙂

Reply:
Sounds fair 👍, let's go with option 2 then? @sahlhoff let's chat later about how to divvy up the work for standardizing the descriptions and updating the guidelines doc.